code/src/pages/Companies.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import BackButton from '../components/BackButton'
+
+const Companies = () => {
+    const [company, setCompany] = useState([])
+    const { comp } = useParams()
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/company/${comp}?api_key=2365aea36d60ef1f206bd1bdf23fd999`)
+        .then((res) => res.json())
+        .then((json) => setCompany(json))
+    },[])
+    return (
+        <>
+            <BackButton />
+            <div className="company-page-container">
+                
+                <div className="company-page">
+                    <h1>{company.name}</h1>
+                    <img className="company-logo" src={company.logo_path === null ? "../assets/company_logo_not_found.png" : `https://image.tmdb.org/t/p/w300${company.logo_path}`} alt={company.name}/>
```

Comment:
Really cool that you did the ternary operator here because not every company has a logo👍